pkg/backup/backup.go

```diff
@@ -116,6 +116,8 @@ func (c *Controller) Backup() error {
 					eventer.EventReasonFailedSetup,
 					err.Error(),
 				)
+			} else {
+				log.Errorf("Failed to write event on %s %s. Reason: %s\n ", restic.Kind, restic.Name, rerr.Error())
```

Comment:
Just use `rerr` instead of `rerr.Error()` . `%s` will take care of it.

Reply:
fixed